tests/integration/_experimental/object_detection/test_evaluator_multiclass_test_sample_metrics.py

```diff
@@ -0,0 +1,5873 @@
+                    ScoredLabeledBoundingBox((41, 41), (42, 42), "c", 0.1),
+                    ScoredLabeledBoundingBox((43, 43), (44, 44), "c", 0.01),
+                    ScoredLabeledBoundingBox((2, 2), (3, 3), "b", 0.8),
+                    ScoredLabeledBoundingBox((4, 4), (5, 5), "e", 0.8),
+                ],
+            ),
+        ),
+        (
+            TestSample(locator=fake_locator(21, "OD")),
+            GroundTruth(
+                bboxes=[
+                    LabeledBoundingBox((21, 21), (22, 22), "e"),
+                    LabeledBoundingBox((23, 23), (24, 24), "e"),
+                    LabeledBoundingBox((25, 25), (26, 26), "e"),
+                    LabeledBoundingBox((27, 27), (28, 28), "e"),
+                    LabeledBoundingBox((31, 31), (32, 32), "e"),
+                    LabeledBoundingBox((33, 33), (34, 34), "e"),
+                    LabeledBoundingBox((35, 35), (36, 36), "e"),
+                    LabeledBoundingBox((37, 37), (38, 38), "e"),
+                    LabeledBoundingBox((41, 41), (42, 42), "e"),
+                    LabeledBoundingBox((43, 43), (44, 44), "e"),
+                ],
+                ignored_bboxes=[
+                    LabeledBoundingBox((2, 2), (3, 3), "b"),
+                    LabeledBoundingBox((4, 4), (5, 5), "b"),
+                    LabeledBoundingBox((6, 6), (7, 7), "b"),
+                    LabeledBoundingBox((8, 8), (9, 9), "b"),
+                    LabeledBoundingBox((1, 1), (2, 2), "a"),
+                    LabeledBoundingBox((3, 3), (4, 4), "a"),
+                    LabeledBoundingBox((5, 5), (6, 6), "a"),
+                    LabeledBoundingBox((7, 7), (8, 8), "a"),
+                ],
+            ),
+            Inference(
+                bboxes=[
+                    ScoredLabeledBoundingBox((21, 21), (22, 22), "b", 0.9),
+                    ScoredLabeledBoundingBox((23, 23), (24, 24), "e", 0.8),
+                    ScoredLabeledBoundingBox((25, 25), (26, 26), "e", 0.7),
+                    ScoredLabeledBoundingBox((27, 27), (28, 28), "e", 0.6),
+                    ScoredLabeledBoundingBox((31, 31), (32, 32), "e", 0.1),
+                    ScoredLabeledBoundingBox((33, 33), (34, 34), "e", 0),
+                    ScoredLabeledBoundingBox((6, 6), (7, 7), "b", 0.9),
+                    ScoredLabeledBoundingBox((8, 8), (9, 9), "b", 0.8),
+                    ScoredLabeledBoundingBox((1, 1), (2, 2), "b", 0.7),
+                    ScoredLabeledBoundingBox((3, 3), (4, 4), "b", 0.6),
+                    ScoredLabeledBoundingBox((5, 5), (6, 6), "b", 0.01),
+                ],
+            ),
+        ),
+    ],
+}
+
+
+TEST_CONFIGURATIONS: Dict[str, ThresholdConfiguration] = {
+    "Threshold: Fixed(0.3), IoU: 0.3, confidence ≥ 0.0": ThresholdConfiguration(
+        threshold_strategy=ThresholdStrategy.FIXED_03,
+        iou_threshold=0.3,
+        min_confidence_score=0.0,
+        with_class_level_metrics=False,
```

Comment:
Are these `single_class` tests necessary for multiclass evaluator test?  Just curious how these differ from the one from [the previous PR](https://github.com/kolenaIO/kolena/pull/107)

Reply:
I think it is still important to keep since a stratification could introduce a test case where only one class is present. Might be pretty rare, but in my opinion, not rare enough to be unnecessary.

Reply:
But wouldn't it still use `with_class_level_metrics=True` configuration on the single class test case?  This is configuration-based and we apply the configuration per test suite. I think a better way to test this out would be construct a test suite with multiple test case and have some of them be single class only and evaluate it with the `with_class_level_metrics=True` configuration.

Reply:
Good point with multiple test cases - will add tests for a test suite with multiple test cases.

A test case with multiple labels can still use `with_class_level_metrics=False` because some people might not use the nested test case metrics feature. I don't think there are identical dups of tests because the `TEST_DATA` is different.

Reply:
Ok I understand now. I think what got me confused was the different interpretation of `single class` and `multiclass` used in tests vs. evaluator config.